tests/testthat/test-shinytest2-teal.R

```diff
@@ -0,0 +1,274 @@
+simple_teal_data <- within(teal_data(), {
+  iris <- iris
+  mtcars <- mtcars
+})
+datanames(simple_teal_data) <- c("iris", "mtcars")
+
+report_module <- function(label = "example teal module") {
```

Comment:
This function is only used once.

Do you want to re-use it in the future (move it to the helper file) or can it be included in the body of the test?